errors/errors.go

```diff
@@ -0,0 +1,99 @@
+package zqe
```

Comment:
I've never seen this before- the pkg name differing from the folder name. Does this mean that goimports will automatically do the right thing?